<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="admin-PC" name="com.Xyphias.QA.RahulCompletePOM.LoginPageTest" tests="3" failures="1" timestamp="06 Aug 2019 09:46:44 GMT" time="85.076" errors="0">
  <testcase name="ValidateLoginPageTitleTest" time="85.076" classname="com.Xyphias.QA.RahulCompletePOM.LoginPageTest">
    <failure type="java.lang.AssertionError" message="Wrong Title expected [ CRMPRO - CRM software for customer relationship management, sales, and support.] but found [CRMPRO - CRM software for customer relationship management, sales, and support.]">
      <![CDATA[java.lang.AssertionError: Wrong Title expected [ CRMPRO - CRM software for customer relationship management, sales, and support.] but found [CRMPRO - CRM software for customer relationship management, sales, and support.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.Xyphias.QA.RahulCompletePOM.LoginPageTest.ValidateLoginPageTitleTest(LoginPageTest.java:59)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- ValidateLoginPageTitleTest -->
  <testcase name="LoginTest" classname="com.Xyphias.QA.RahulCompletePOM.LoginPageTest">
    <skipped/>
  </testcase> <!-- LoginTest -->
  <testcase name="VerifyLogoTest" classname="com.Xyphias.QA.RahulCompletePOM.LoginPageTest">
    <skipped/>
  </testcase> <!-- VerifyLogoTest -->
</testsuite> <!-- com.Xyphias.QA.RahulCompletePOM.LoginPageTest -->
